### PR TITLE
Adjustments to new HRW

### PR DESCRIPTION
**Pull request by IvarsKarpics**

### CODE REVIEW

File: gui/bricks/ImageTrackingStatusBrick.py [hunk 1]
```diff
@@ -33,6 +33,7 @@ class ImageTrackingStatusBrick(BaseWidget):
         "tracking": Colors.LIGHT_GREEN,
         "disabled": Colors.LIGHT_GRAY,
         "error": Colors.LIGHT_RED,
+        "tracking": Colors.LIGHT_GREEN,
```
Comment (rhfogh) [inline gui/bricks/ImageTrackingStatusBrick.py]: Is the duplication deliberate?

File: gui/widgets/pyqtgraph_widget.py [hunk 2]
```diff
@@ -26,6 +26,9 @@
 __license__ = "LGPLv3+"
 
 
+#pg.setConfigOption('background', 'w')
+
+
 class TwoDimenisonalPlotWidget(QtImport.QWidget):
```
Comment (rhfogh) [inline gui/widgets/pyqtgraph_widget.py]: Is that the right spelling? 'Twodimeniso...'

**Comment by rhfogh** [comment]
OK to merge - just look at the two comments first.

File: gui/bricks/ImageTrackingStatusBrick.py (hunk 1 above)
Comment (IvarsKarpics) [reply gui/bricks/ImageTrackingStatusBrick.py]: no. fixed.

File: gui/widgets/pyqtgraph_widget.py (hunk 2 above)
Comment (IvarsKarpics) [reply gui/widgets/pyqtgraph_widget.py]: I changed name to PlotWidget.

**Comment by jordiandreu** [comment]
It is fine for me. It overlaps with some small bugs I found when trying to run the mockup/qt4 application. Once merged, I will upload a new PR adding some little changes.

**Pull request merged**